pandas/core/groupby/groupby.py

```diff
@@ -923,7 +923,7 @@ def _make_wrapper(self, name: str) -> Callable:
             # as are not passed directly but in the grouper
             f = getattr(self._obj_with_exclusions, name)
             if not isinstance(f, types.MethodType):
-                return self.apply(lambda self: getattr(self, name))
+                return cast(Callable, self.apply(lambda self: getattr(self, name)))
```

Comment:
can you include the mypy error as a comment

Reply:
fixed in https://github.com/pandas-dev/pandas/pull/45904/commits/ddb824d38dc17bb4413bc8e01dad24eadf96a94b